src/App.tsx

```diff
@@ -1,61 +1,120 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './type/Todo';
+import { User } from './type/User';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+let maxId = Math.max(...todos.map(todo => todo.id));
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('');
```

Comment:
I expect this variable to be an object from its name, but you store the id here.
```suggestion
  const [selectedUserId, setSelectedUserId] = useState('');
```